hs_core/views/__init__.py

```diff
@@ -1819,10 +1827,36 @@ def create_user_group(request, *args, **kwargs):
     return HttpResponseRedirect(request.META["HTTP_REFERER"])
 
 
+@login_required
+def update_user_community(request, community_id, *args, **kwargs):
+    """Updates metadata for a community"""
+
+    community_to_update = hydroshare.utils.community_from_id(community_id)
+
+    # TODO: need community equivalent of can_change_group_flags
```

Comment:
this TODO seems pretty important--don't we want to implement that before we merge this PR?

Reply:
This is no longer needed as an exception handler was added right below. I'll delete this comment.